src/view/transaction-detail.js

```diff
@@ -18,7 +18,16 @@ const TransactionDetailView = ({ store, nav }) => (
   <Background color={color.blackDark}>
     <MainContent style={styles.content}>
       <Modal title="Transaction Details" onClose={() => nav.goTransactions()}>
-        <DetailField name="ID">{store.selectedTransaction.id}</DetailField>
+        {store.selectedTransaction.confirmationsLabel ? (
+          <DetailField name="Transaction ID">
+            {store.selectedTransaction.id}
+          </DetailField>
+        ) : null}
+        {store.selectedTransaction.hash ? (
+          <DetailField name="Invoice ID">
+            {store.selectedTransaction.hash}
+          </DetailField>
+        ) : null}
```

Comment:
Could this be simplified to:
```
        {store.selectedTransaction.confirmationsLabel ? (
          <DetailField name="Transaction ID">
            {store.selectedTransaction.id}
          </DetailField>
        ) : (
          <DetailField name="Invoice ID">
            {store.selectedTransaction.hash}
          </DetailField>
        )}

```
since won't a transaction either be on-chain *or* an invoice?